pyfactorybridge/__init__.py

```diff
@@ -34,19 +36,47 @@ def __init__(self, address, password=None, token=None):
         else:
             logging.error("No password provided, some functions may not work.")
 
-    def __request(self, function, properties={}) -> dict[Any]:
+    def __build_request_data(
+        self, function: str, properties: dict | None = None
+    ) -> None:
```

Comment:
`-> dict`

Reply:
Ah yes, I first only made this function use side-effects but changed it down the line and forgot that.